city_scrapers/spiders/tulok_unionps.py

```
@@ -0,0 +1,79 @@
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
```

Ruff (0.14.6)

Unused method argument: `item`

(ARG002)